client/src/services/hotkeyManager.ts

```diff
@@ -0,0 +1,126 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { isDesktop, isLinux, isMacOS, isMobile, isWeb, isWindows } from '@/parsec/environment';
+
+export const HotkeyManagerKey = 'hotkey';
+
+export enum Modifiers {
+  Ctrl = 1 << 0,
+  Alt = 1 << 1,
+  Shift = 1 << 2,
+}
+
+export enum Platforms {
+  Windows = 0,
+  MacOS = 1 << 0,
+  Linux = 1 << 1,
+  Web = 1 << 2,
+  Mobile = 1 << 3,
+  Desktop = Windows | MacOS | Linux,
+  All = Desktop | Web | Mobile,
+}
+
+export interface Hotkey {
+  key: string;
+  modifiers: Modifiers;
+  platforms: Platforms;
```

Comment:
`platforms` should be a number. For example, `Linux | MacOS` would take the value `1 + 2` which is not a valid `Platforms` type.